src/chrome/komodo/content/bindings/views-buffer.p.xml

```diff
@@ -1494,11 +1497,17 @@
                 {
                     // Get codeintel to re-scan the document - but delay the
                     // scan for repeated typing - bug 101502.
-                    if (linesAdded) {
-                        ko.codeintel.scan_document(koDoc, linesAdded, true /* forcedScan */);
-                    } else {
+                    // Only run if current timeout can be cancelled.  When a new line is added, we'll mark it to not be cancelled.
+                    if (this.codeintel_scan_timeout_cancelable)
+                    {
+                        // Don't allow cancelling if lines were added or removed
+                        this.codeintel_scan_timeout_cancelable = (linesAdded != 0);
```

Comment:
Shouldn't this be `linesAdded == 0` if we want it to _not_ be cancellable when lines have been added?

Reply:
Yes, thanks.  I fixed that.